.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: node_js
 node_js:
     - "6"
+addons:
+  sauce_connect: true
```

Comment:
This is sauce labs travis integration. We set our login info in our travis environment variables so we don't have to expose them here, and they won't be logged.